lib/migration.js

```diff
@@ -189,6 +189,59 @@ function mixinMigration(MsSQL) {
     return statements;
   };
 
+  MsSQL.prototype.generatePropIndexName = function(propName, model) {
+    var i = model.properties[propName].index;
```

Comment:
`var i = model.properties[propName] && model.properties[propName].index;`

Just in case the property specified by propName doesn't exist.
I'm aware that the calling function will never pass in a propName that isn't defined, given that it's sourced from the list of props previously filtered, but someone could change this in the future and it's a small change to protect ourselves.